src/components/PageHeader.tsx

```diff
@@ -168,17 +177,41 @@ export const DashboardPageHeader = () => {
         console.log('Selected: ', activeItem);
         setActiveItem(result.itemId);
     };
+    const [
+        getAuthZ,
+        { loading: _authzLoading, error: _authzError, data: authzData },
+    ] = useLazyQuery<AuthZMapping>(AuthZMappingQuery, {
+        errorPolicy: 'all',
+        /* need to re-fetch each time when user press save/back button */
+        fetchPolicy: 'network-only',
```

Comment:
I do not understand your main concern here.
In react evrything is flickering. Literally evrything.
Each component. Many times, for each action.
I cannot use other `fetchPolicy`, since I need a browser sends a network request, and ignore cache.

Reply:
I wish I could record a video, but I couldn't get the recording working right now. I'll try later.

Reply:
Work item: check if we can used cached query.